scripts/tokenomics-emulation/config.json

```diff
@@ -31,12 +31,14 @@
     }
   ],
   "nodes_amount": 4,
+  "batch_count": 2,
```

Comment:
It is 2 levels. Put it to the emulation object.

Reply:
Moved in emulation